src/umpire/Umpire.cpp

```diff
@@ -136,9 +136,8 @@ std::size_t get_process_memory_usage_hwm()
 
     if (key == "VmHWM:") {
       std::size_t resident_hwm;
-      long page_size{::sysconf(_SC_PAGE_SIZE)};
       ss >> resident_hwm;
-      rval = std::size_t{resident_hwm * page_size};
+      rval = std::size_t{resident_hwm * 1024};
```

Comment:
Can you add a comment here about the magical 1024?

Reply:
@davidbeckingsale, this is done (I need your review again)